files/pod-template-file.yaml

```diff
@@ -26,11 +26,19 @@ spec:
     - args: []
       name: "base"
       command: []
+      {{/* Environment section duplicated from _helpers.yaml file */}}
+      {{- if .Values.mountAllFromSecretName }}
```

Comment:
This variable, `mountAllFromSecretName`, is vague. As part of this change can we add a comment in `values.yaml` documenting what this variable is for?

Reply:
done